services/QuillLMS/spec/models/concept_feedback_spec.rb

```diff
@@ -74,39 +75,116 @@
       it 'adds the translations to the data' do
         expect(subject["translatedDescription"]).to eq(translation)
       end
+
+      context 'there is only a gengo translation' do
+        let(:translated_text) { create(:gengo_translated_text, translation: translation)}
+
+        it 'adds the translations to the data' do
+          expect(subject["translatedDescription"]).to eq(translation)
+        end
+      end
+
+      context 'there is a gengo and open ai translation' do
```

Comment:
Take or leave:

I mentioned this in the relevant files, but I'm not sure this is a case we need to try to solve right now.

Reply:
This is a case that I run into a lot on my local machine because I am testing them both. I think we can take it out later if it turns out we get rid of Gengo, but since the code is already there, and it's useful for dev purposes, I'm gonna leave it in for now.